ts/config.ts

```diff
@@ -32,6 +32,7 @@ const DEFAULT_BACKGROUND_ACTIVITY_TIMEOUT_S = 30;
 // Default number of workers to fetch message.
 const DEFAULT_TOT_MESSAGE_FETCH_WORKERS = 5;
 
+export const tosVersion: number = Config.TOS_VERSION;
```

Comment:
you should use `t.Integer.decode` (see below) or (even better) [NonNegativeInteger](https://github.com/teamdigitale/io-ts-commons/blob/master/src/numbers.ts#L86)

Reply:
@chiadav carefully read my statement here

Reply:
you should use `NonNegativeInteger.decode` - NOT casting!